lib/fake_data_loader.rb

```diff
@@ -62,7 +62,10 @@ def self.load_sessions
           :feedback_session,
           provider: user,
           receiver: User.where.not(id: user.id).sample
-        )
+        ).tap do |session|
+          create_list(:comment, rand(1..5), feedback_session: session, body: Faker::Lorem.paragraph(sentence_count: 2, supplemental: true,
+                                                                                                    random_sentences_to_add: 4))
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Layout/LineLength: Line is too long. [128/100]